### ISSUE TITLE
Enhancement: Allow SMS/MMS to be sent/received in the app.

### ISSUE THREAD

**Issue by IdrissDimson** [bboyairwreck/PieMessage#22]
It would be cool to have PieMessage become an all-in-one messaging app. In addition to doing this, we would also need to have SMS/MMS chats a different color than the iMessage chats.

**Comment by ghobs91** [comment]
Agreed! Ideally regular SMS/MMS and iMessages with the same contact would be merged into the same thread, rather than separate conversations.
